netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -146,7 +155,8 @@ static NettyClientHandler newHandler(Http2Connection connection,
 
     // Create the local flow controller configured to auto-refill the connection window.
     connection.local().flowController(new DefaultHttp2LocalFlowController(connection,
-            DEFAULT_WINDOW_UPDATE_RATIO, true));
+        DEFAULT_WINDOW_UPDATE_RATIO , true));
```

Comment:
two things: 
- extra space after comma,
- lines should be broken at the highest possible semantic level.  In this case the `connection` variable should be on the same line as the other arguments.